unit test/addItem.test.js

```diff
@@ -0,0 +1,60 @@
+/**
+ * @jest-environment jsdom
+ */
+
+// eslint-disable-next-line import/extensions
+import createMockDocument from './mocks/documentMock.js';
+// eslint-disable-next-line import/no-unresolved, import/extensions
+import { getAddedTodos } from '../src/modules/newTask';
+// eslint-disable-next-line import/no-unresolved, import/extensions
+import { todo } from '../src/modules/displayTask';
```

Comment:
- [ Required ] It would be better if you don't disable `eslint` linter checks, linter configurations help you as a developer to write clean and professional code. Please consider removing the  statements that are disabling some linter checks in all of your `.js` files.

- Your import statements should look as the example shown below.

```JavaScript
import createMockDocument from './mocks/documentMock';
import { getAddedTodos } from '../src/modules/newTask';
import { todo } from '../src/modules/displayTask';
```